frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -647,12 +647,18 @@ function ReaderBookmark:onGotoNextBookmark(pn_or_xp)
     return true
 end
 
-function ReaderBookmark:onGotoNextBookmarkFromPage()
+function ReaderBookmark:onGotoNextBookmarkFromPage(add_current_location_to_stack)
+    if add_current_location_to_stack ~= false then -- nil or true
```

Comment:
Isn't this more appropriate?
```lua
if not add_current_location_to_stack then
```

Reply:
I want `nil` (what other code I won't touch will provide) to behave as `true`. Only in the explicite places I want to not have that default behaviour to happen, I'll provide `false`.
(We use that kind of construct to have a positive-named parameter in other places.)